projects/admin/src/app/record/detail-view/entities-detail-view/remote/entities-remote-detail-view.component.html

```diff
@@ -0,0 +1,47 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2019-2023 RERO
+  Copyright (C) 2019-2023 UCLouvain
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<ng-container *ngIf="record$ | async as record">
+  <ng-container>
+    <div class="float-right mt-2 mb-4 mr-n4">
+      <a (click)="search(record.metadata)" class="btn btn-sm btn-primary">
+        <i class="fa fa-search" aria-hidden="true"></i>
+        {{ 'Search documents' | translate }}
+      </a>
+    </div>
+
+    <h1 class="mb-0">{{ record.metadata | extractSourceField : 'authorized_access_point' }}</h1>
+    <small *ngVar="icon(record.metadata.type) as icon">
+      <i class="fa {{ icon.class }}" title="{{ icon.title }}" aria-hidden="true"></i>
+      {{ 'MEF ID' | translate }}: {{ record.metadata.pid }}
+    </small>
+
+    <!-- SOURCES -->
+    <article class="card my-2" *ngFor="let source of record.metadata.sources.sort()">
+      <header class="card-header">
+        <b class="card-title mb-0">
+          {{ source | translate | uppercase }}
+        </b>
+      </header>
+      <article id="contribution-{{ source }}" class="card-body collapse show" [ngSwitch]="record.metadata.type">
+        <admin-remote-entities-person-detail-view *ngSwitchCase="entityType.PERSON" [record]="record.metadata[source]" [source]="source"></admin-remote-entities-person-detail-view>
+        <admin-remote-entities-organisation-detail-view *ngSwitchCase="entityType.ORGANISATION" [record]="record.metadata[source]" [source]="source"></admin-remote-entities-organisation-detail-view>
+        <span *ngSwitchDefault>Missing contribution type.</span>
```

Comment:
Maybe adapt the message... it's not really that entity type is missing but not yet implemented